README.md

```diff
@@ -1175,7 +1186,32 @@ POST /policies
       "service-offerings": [
         "p-config-server"
       ]
-    }
+    },
+    {
+    "resource-notification-policies": [
+        {
+            "resource-email-template": {
+                "from": "admin@pcf.demo.ironleg.me",
+                "subject": "Platform Updates",
+                "body": "Please take a moment to review the platform updates and share it with your Org users"
+            },
+            "resource-email-metadata": {
+                "resource": "organizations",
```

Comment:
I like that you limited metadata definition to one resource type.

Reply:
Are you sure that the resource type is plural form?  Or is it really singular form? E.g., instead of `organizations` would it be `organization`?

Reply:
I see it's actually what's allowed in `ResourceType` enum.  You're good in this example.